packages/components/src/internal/components/search/FilterCards.tsx

```diff
@@ -49,7 +49,7 @@ export const FilterCard: FC<FilterEditProps> = memo(props => {
                 <div className={'filter-card__header ' + entityDataType.filterCardHeaderClass}>
                     <div className="pull-left">
                         <div className="secondary-text">{capitalizeFirstChar(entityDataType.nounAsParentSingular)}</div>
-                        <div className="primary-text">{schemaQuery.queryName}</div>
+                        <div className="primary-text">{dataTypeDisplayName}</div>
```

Comment:
any reason to fall back on using the `schemaQuery.queryName` here and below (i.e. `dataTypeDisplayName ?? schemaQuery.queryName`)? I assume not, but just checking since it looks like `dataTypeDisplayName` is an optional prop.